.env.sample

```
@@ -76,6 +76,13 @@ ERROR_ALERT_COOLDOWN=5m
 # 중복 억제 캐시 정리 주기(ms). 기본 5분, 통상 조정 불필요
 NOTIFICATION_DEDUP_CLEANUP_INTERVAL_MS=300000
 
+# Redis 키 누수 정리 스케줄러 (6h 간격, 04:30/10:30/16:30/22:30 KST)
+REDIS_CLEANUP_ENABLED=true
+REDIS_CLEANUP_SCAN_BATCH_SIZE=500
+# N일 이상 오래된 ranking/ranking_detail 키 삭제 (자정 스케줄러 영역 보호 위해 2 이상 권장)
+REDIS_CLEANUP_PURGE_OLDER_THAN_DAYS=2
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REDIS_CLEANUP_PURGE_OLDER_THAN_DAYS key should go before the REDIS_CLEANUP_SCAN_BATCH_SIZE key

(UnorderedKey)